@adriaanm there are some scary looking lines in the publish-core log, but i guess just a part of checking if the revision has been published before?

```
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
++ grep -i error /tmp/pr-scala.1GVp/tmp/mvn.log
[ERROR] Failed to execute goal on project typesafeDummy: Could not resolve dependencies for project com.typesafe:typesafeDummy:war:1.0-SNAPSHOT: Could not find artifact org.scala-lang:scala-reflect:jar:2.12.0-595e23e-SNAPSHOT in sonatype.snapshot (https://oss.sonatype.org/content/repositories/snapshots) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

--- a/test/files/scalacheck/redblacktree.scala
+++ b/test/files/scalacheck/redblacktree.scala
@@ -33,7 +33,7 @@ package scala.collection.immutable.redblacktree {
 
     def mkTree(level: Int, parentIsBlack: Boolean = false, label: String = ""): Gen[Tree[String, Int]] =
       if (level == 0) {
-        value(null)
+        const(null)
       } else {
         for {
           oddOrEven <- choose(0, 2)

--- a/versions.properties
+++ b/versions.properties
@@ -13,21 +13,24 @@ starr.version=2.11.6
 # These are the versions of the modules that go with this release.
 # These properties are used during PR validation and in dbuild builds.
 
-# The scala.binary.version determines how modules are resolved. After 2.x.0 is released,
-# the binary version is 2.x. For milestones and RCs, modules are cross-built against the
-# full version, so the value, so the value is the full version (e.g. 2.12.0-M1).
+# The scala.binary.version determines how modules are resolved. For example, it
+# determines which partest artifact is being used for running the tests.
+# It has to be set in the following way:
+#  - After 2.x.0 is released, the binary version is 2.x.
+#  - During milestones and RCs, modules are cross-built against the full version.
+#    So the value is the full version (e.g. 2.12.0-M1).
 scala.binary.version=2.11
 
 # external modules shipped with distribution, as specified by scala-library-all's pom
-scala-xml.version.number=1.0.3
-scala-parser-combinators.version.number=1.0.3
-scala-swing.version.number=1.0.1
-akka-actor.version.number=2.3.4
+scala-xml.version.number=1.0.4
+scala-parser-combinators.version.number=1.0.4
+scala-swing.version.number=1.0.2
+akka-actor.version.number=2.3.10
 jline.version=2.12.1
 
 # external modules, used internally (not shipped)
-partest.version.number=1.0.6
-scalacheck.version.number=1.11.4
+partest.version.number=1.0.7
+scalacheck.version.number=1.12.2
 
 # TODO: modularize the compiler
 #scala-compiler-doc.version.number=1.0.0-RC1

